Project.toml

```diff
@@ -0,0 +1,12 @@
+name = "AdvancedMH"
+uuid = "5b7e9947-ddc0-4b3f-9b55-0d8042f74170"
+version = "0.0.1"
+
+[deps]
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+MCMCChains = "c7f686f2-ff18-58e9-bc7b-31028e88f75d"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Turing = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
Maybe consider remove the dependency on `Zygote`? Since MH sampler does not require gradient information